rs/boundary_node/anonymization/backend/src/lib.rs

```diff
@@ -1,2 +1,118 @@
-// I put this here for CI, will remove on the next PR
-pub const TMP: u32 = 0;
+use std::{cell::RefCell, time::Duration};
+
+use anonymization_interface::{
+    self as ifc, InitArg, QueryResponse, RegisterResponse, SubmitResponse,
+};
+use candid::Principal;
+use ic_cdk::{id, spawn};
+use ic_cdk_timers::set_timer_interval;
+use ic_stable_structures::{
+    memory_manager::{MemoryId, MemoryManager, VirtualMemory},
+    DefaultMemoryImpl, StableBTreeMap,
+};
+use lazy_static::lazy_static;
+use registry::{Client, List};
+
+mod registry;
+
+type Memory = VirtualMemory<DefaultMemoryImpl>;
+
+type StableMap<K, V> = StableBTreeMap<K, V, Memory>;
+type StableSet<T> = StableMap<T, ()>;
+
+thread_local! {
+    static MEMORY_MANAGER: RefCell<MemoryManager<DefaultMemoryImpl>> =
+        RefCell::new(MemoryManager::init(DefaultMemoryImpl::default()));
+}
+
+const REGISTRY_CANISTER_ID: &str = "bnz7o-iuaaa-aaaaa-qaaaa-cai";
```

Comment:
Should this be an InitArg? On mainnet it is `rwlgt-iiaaa-aaaaa-aaaaa-cai`

Reply:
IMO we should import [this](https://github.com/dfinity/ic/blob/9f4e6dd68c30f494ded605fa49b2a4d50d72a790/rs/nns/constants/src/lib.rs#L75) constant.